packages/dcos-integration-test/extra/test_mesos.py

```diff
@@ -181,6 +181,22 @@ def get_ps_output():
         assert len(get_ps_output().split()) <= 4, 'UCR app has more than 4 processes running in its pid namespace'
 
 
+def test_memory_profiling(dcos_api_session):
+    # Test that we can fetch raw memory profiles
+    master_ip = dcos_api_session.masters[0]
+    r0 = dcos_api_session.get(
+        '/memory-profiler/start', host=master_ip, port=5050)
+    assert r0.status_code == 200
```

Comment:
We should print the body on assertion failures, here and below.

    assert r0.status_code == 200, r0.text

Reply:
What do you expect to see in the body? "File Not Found"? Any interesting output would be in the web server logs.

Reply:
A request could file at a number of places as it finds its way to the master. Some of these response return additional data on top of the status code. Outputting that (sometimes redundant) text here can help genug issues faster.

We also do that in a couple of places in this test already, uniformity yadadada.

Reply:
Ok, done.